src/mpid/ch4/shm/posix/posix_init.h

```diff
@@ -78,39 +78,41 @@ static inline int MPIDI_POSIX_mpi_init_hook(int rank, int size, int *n_vnis_prov
     mpi_errno =
         MPIDU_shm_seg_alloc(MAX
                             ((num_local * ((num_local - 1) * sizeof(MPIDI_POSIX_fastbox_t))),
-                             MPIDI_POSIX_ASYMM_NULL_VAL), (void **) &fastboxes_p, MPL_MEM_SHM);
+                             MPIDI_POSIX_ASYMM_NULL_VAL), MPIR_MEMKIND__DRAM,
+                            (void **) &fastboxes_p, MPL_MEM_SHM);
```

Comment:
I'm probably missing some context here.  Is this being forced to `DRAM` instead of checking for the user preference?  This happens more times below.  Some of them are non-cell/fastbox allocations, which are fine.  But thought we are allowing for multiple memory types for fbox and cells.

Reply:
This should be addressed now